server/__test__/index.test.js

```diff
@@ -229,9 +232,51 @@ describe('authentication', () => {
 
     test('should get news from the news table', async () => {
       const { rows } = await getNews();
-      return expect(rows[5].content).toEqual(
+      const allContent = rows.map((row) => row.content);
+      return expect(allContent).toContain(
```

Comment:
Cool, thanks for making this change!

Reply:
it was much easier than checking what's right, especially that this news will be changed and their order will change for sure.